interface/src/app/node/node_search.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * "adage.node_search" module.
+ */
+
+angular.module('adage.node_search', [
```

Comment:
Quick question - should we use underscore when naming this module rather than camelCase? Do camelCase rules in javascript apply to module names as well?